src/index.html

```diff
@@ -13,5 +13,150 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="#">
+      <fieldset >
+        <legend>Personal infotmation</legend>
+        <div>
+          <label for="surname">Surname: </label>
+          <input
+            type="text" 
+            id="surname" 
+            required
+            autocomplete="off"
+          >
+        </div>
+        <div>
```

Comment:
add one empty line between siblings that have the same tag name. check throughout your code